website/content/middleware/jwt.md

```diff
@@ -27,50 +27,92 @@ e.Use(middleware.JWTWithConfig(middleware.JWTConfig{
 }))
 ```
 
+*Usage with different JWT library/repository*
+
+> NB: Due backwards compatibility reasons Echo still uses unmaintained, with potential security risks `dgrijalva/jwt-go` version. We recommend using maintained fork ala https://github.com/golang-jwt/jwt
```

Comment:
I'd prefer:

Note: To be backwards compatible Echo still uses the currently unmaintained JWT implementation `dgrijalva/jwt-go`, which has known potential security risks. We strongly recommend using a maintained fork like https://github.com/golang-jwt/jwt.